packages/webapp/components/layouts/AccountLayout/Security/index.tsx

```diff
@@ -24,14 +34,56 @@ interface AccountSecurityDefaultProps {
   isEmailSent?: boolean;
   onSwitchDisplay: (display: Display) => void;
 }
+export interface ManageSocialProvidersProps {
+  type: 'link' | 'unlink';
```

Comment:
I think we can create a dedicated type for this and export it so the component requiring it as props would also be type-strict.